agriculture/data/website_view.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="contactus" model="ir.ui.view">
+        <field name="name">Contact Us</field>
+        <field name="key">agriculture.contactus</field>
+        <field name="type">qweb</field>
+        <field name="website_id" ref="website.default_website"/>
+        <field name="arch" type="xml">
+            <t name="Contact Us" t-name="website.contactus">
+                <t t-call="website.layout">
+                    <t t-set="logged_partner" t-value="request.env['website.visitor']._get_visitor_from_request().partner_id"/>
+                    <t t-set="contactus_form_values" t-value="{'email_to': res_company.email, 'name': request.params.get('name', ''), 'phone': request.params.get('phone', ''), 'email_from': request.params.get('email_from', ''), 'company': request.params.get('company', ''), 'subject': request.params.get('subject', ''),}"/>
```

Comment:
Could you make sure it is linked to the CRM?

Reply:
![image](https://github.com/odoo/industry/assets/120459793/6cc64ebe-f51f-425e-b904-f6329950c882)


It is linked to crm.lead

Reply:
But when you submit the form, it doesn't create a lead

Reply:
I am able to generate the crm lead
https://www.awesomescreenshot.com/video/28835367?key=635ae17bab4e0ed357ef4461e750e222

Reply:
I was tricked by the filter :man_facepalming: sorry about that